### PR TITLE
Rework Charts to TS

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

**Review by pomahtri — CHANGES_REQUESTED** [review]
If you want to save correct file history (to use blame), you need to merge this PR without squashing. In this case all commits will be included in main branch as is, so please change their names, and remove merge commits

**Comment by alexanderPolosatov** [comment]
> If you want to save correct file history (to use blame), you need to merge this PR without squashing. In this case all commits will be included in main branch as is, so please change their names, and remove merge commits

ok. thank you

File: testing/tests/DevExpress.common/charts.tests.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { registerPattern, registerGradient, getGraphicObjects } from 'common/charts';
+import { registerPattern, registerGradient } from 'common/charts';
+import graphicObjects from '__internal/common/m_charts';
```
Comment (Krijovnick) [inline testing/tests/DevExpress.common/charts.tests.js:2]: Could we import registerPattern and registerGradient from '__internal/common/m_charts'?

Comment (alexanderPolosatov) [reply testing/tests/DevExpress.common/charts.tests.js:2]: common/charts - it's a public path to this methods. I think we should testing this methods using public paths(or how we check, what public methods is working)

Comment (Krijovnick) [reply testing/tests/DevExpress.common/charts.tests.js:2]: ok, could we import { getGraphicObjects } there instead of graphicObjects?

Comment (alexanderPolosatov) [reply testing/tests/DevExpress.common/charts.tests.js:2]: we have
export default { getGraphicObjects };
this kind of export was added for [mock](https://github.com/DevExpress/DevExtreme/blob/23_2/testing/tests/DevExpress.viz.charts/pieChart.tests.js#L2438) this function. if we will not export { getGraphicObjects }, we can't write this mock.

**Review by Krijovnick — APPROVED** [review]

**Review by marker-dao — APPROVED** [review]

**Pull request closed without merging**